docs/glossary.rst

```diff
@@ -0,0 +1,60 @@
+Voxa Documentation!
```

Comment:
Yes, Mitchel wanted to have the index on the glossary. If i just added this index to the glossary it shows twice every item